operations/patch_util.go

```diff
@@ -621,17 +621,22 @@ Continue?`, uncommittedChangesFlag), true), nil
 // If no dir is provided, we use the current working directory.
 // The branch argument is used to determine where to generate the merge base from, and any extra
 // arguments supplied are passed directly in as additional args to git diff.
-func loadGitData(dir, branch, ref, commits string, format bool, extraArgs ...string) (*localDiff, error) {
-	// branch@{upstream} refers to the branch that the branch specified by branchname is set to
-	// build on top of. This allows automatically detecting a branch based on the correct remote,
-	// if the user's repo is a fork, for example. This also works with a commit hash, if given.
+func loadGitData(dir, remote, branch, ref, commits string, format bool, extraArgs ...string) (*localDiff, error) {
+	if remote == "" {
+		remote = "upstream"
+	}
+	// remote/branch refers directly to the remote branch and does not require a local branch.
+	// branch@{remote} referes to a local branch that has a tag of a remote branch attached to it.
```

Comment:
nit: refers*

Reply:
Since git tags are a separate concept, can we use the original wording or add a little bit for the remote branch? The [git docs](https://git-scm.com/book/en/v2/Git-Branching-Remote-Branches) (see `Upstream shorthand`) describe branch@{upstream} as a specific syntactic sugar in git

Reply:
I did not know this wasn't referencing a git tag! I got a little confused because I was thinking that users probably dont tag their branches with the remote they track- and I was thinking if it's a process we enforce. Thank you for linking the docs for that, helped a lot!